lib/parser/livesplit_core_parser.rb

```diff
@@ -62,91 +79,82 @@ def self.parse(run, fast: false)
     run_object[:total_playtime_ms] = total_playtime.total_seconds * 1000
     total_playtime.dispose
 
-    unless fast
-      run_object[:history] = []
-      run_object[:indexed_history] = {}
-      run_object[:realtime_history] = []
-      (0...run.attempt_history_len).each do |i|
-        attempt = run.attempt_history_index(i)
-        attempt_id = attempt.index.to_i
-        time = attempt.time
-
-        # See https://github.com/glacials/splits-io/pull/474/files#r241242051
-        attempt_started = attempt.started
-        attempt_ended = attempt.ended
-
-        run_object[:history] << {
-          attempt_number:       attempt_id,
-          realtime_duration_ms: time.real_time.try(:total_seconds).try(:*, 1000) || 0,
-          gametime_duration_ms: time.game_time.try(:total_seconds).try(:*, 1000) || 0,
-
-          started_at: attempt_started && DateTime.parse(attempt_started.to_rfc3339),
-          ended_at:   attempt_ended   && DateTime.parse(attempt_ended.to_rfc3339)
-        }
+    run_object[:history] = []
+    (0...run.attempt_history_len).each do |i|
+      attempt = run.attempt_history_index(i)
+      attempt_id = attempt.index.to_i
+      time = attempt.time
 
-        # See https://github.com/glacials/splits-io/pull/474/files#r241242051
-        [attempt_started, attempt_ended].compact.each(&:dispose)
+      # See https://github.com/glacials/splits-io/pull/474/files#r241242051
+      attempt_started = attempt.started
+      attempt_ended = attempt.ended
 
-        if time.real_time.try(:total_seconds).present?
-          run_object[:indexed_history][attempt_id] = time.real_time.try(:total_seconds)
-          run_object[:realtime_history] << time.real_time.try(:total_seconds)
-        end
-      end
-    end
+      run_object[:history] << {
+        attempt_number:       attempt_id,
+        realtime_duration_ms: time.real_time.try(:total_seconds).try(:*, 1000) || 0,
+        gametime_duration_ms: time.game_time.try(:total_seconds).try(:*, 1000) || 0,
 
-    (0...run.len).each do |i|
-      segment = run.segment(i)
-      split = Split.new
-      split.name = segment.name
+        started_at: attempt_started && DateTime.parse(attempt_started.to_rfc3339),
+        ended_at:   attempt_ended   && DateTime.parse(attempt_ended.to_rfc3339),
 
-      split.realtime_end = segment.personal_best_split_time.real_time.try(:total_seconds) || 0
-      split.realtime_duration = [0, split.realtime_end - run_object[:realtime_time]].max
-      split.realtime_start = split.realtime_end - split.realtime_duration
+        pause_duration_ms: attempt.pause_time.try(:total_seconds).try(:*, 1000) || 0
```

Comment:
For timers that don't track pause time, I'd rather this be `nil` than `0`, so we don't misrepresent runs as having not paused when they had.